site/src/examples/segmented-button-group/Variants.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from "react";
+import { Button, StackLayout } from "@salt-ds/core";
+import { SegmentedButtonGroup } from "@salt-ds/lab";
+
+export const Variants = (): ReactElement => (
+  <StackLayout>
+    <SegmentedButtonGroup>
+      <Button>Button</Button>
```

Comment:
nit. maybe we could make the buttons a bit more realistic like the ones in button (search, send, refresh.... or something like that?